Cargo.toml

```diff
@@ -93,6 +93,10 @@ path = "examples/lua/function_params.rs"
 name = "hello_world_lua"
 path = "examples/lua/hello_world.rs"
 
+[[example]]
+name = "multiple_plugins"
```

Comment:
The other examples follow a  naming convention `XXX_languagename` so to match this this example would be named `multiple_plugins_lua`

Reply:
Resolved